docs/pt/docs/tutorial/path-params.md

```diff
@@ -0,0 +1,246 @@
+# Parâmetros de rota
+
+Você pode declarar "parâmetros" ou "variáveis" de rota com a mesma sintaxe usada para formatar strings em Python:
+
+```Python hl_lines="6-7"
+{!../../../docs_src/path_params/tutorial001.py!}
+```
+
+
+O valor do parâmetro de rota `item_id` será passado para sua função como o argumento `item_id`.
+
+Então, se você executar este exemplo e for para <a href="http://127.0.0.1:8000/items/foo" class="external-link" target="_blank">http://127.0.0.1:8000/items/foo</a>, você verá a seguinte resposta:
+
+```JSON
+{"item_id":"foo"}
+```
+
+## Parâmetros de rota com tipos
+
+Você pode declarar o tipo de um parâmetro de rota na função, usando anotações de tipo padrão de Python:
+
+```Python hl_lines="7"
+{!../../../docs_src/path_params/tutorial002.py!}
+```
+
+Neste caso, `item_id` é declarado como um `int`.
+
+!!! check
+    Isso lhe dará suporte ao editor dentro de sua função, com verificações de erros, preenchimento automático, etc.
+
+## <abbr title="também conhecido como: serialização, parseamento, marshalling">Conversão</abbr> de Dados
+
+Se você executar este exemplo e abrir o seu navegador em <a href="http://127.0.0.1:8000/items/3" class="external-link" target="_blank">http://127.0.0.1:8000/items/3</a>, você verá a seguinte resposta:
+
+```JSON
+{"item_id":3}
+```
+
+!!! check
+   Observe que o valor que sua função recebeu (e devolveu) é `3`, como o tipo Python `int`, não uma string `"3"`.
+
+    Então, com essa declaração de tipo, **FastAPI** oferece a você uma <abbr title="convertendo a string que vem de uma requisição HTTP em dados Python">"conversão"</abbr> automática de requisição.
```

Comment:
This check is not showing correctly on the preview website. 

I believe the following suggestion will fix it.
```suggestion
!!! check
    Observe que o valor que sua função recebeu (e devolveu) é `3`, como o tipo Python `int`, não uma string `"3"`.

    Então, com essa declaração de tipo, **FastAPI** oferece a você uma <abbr title="convertendo a string que vem de uma requisição HTTP em dados Python">"conversão"</abbr> automática de requisição.
```